samples/excel/85-preview-apis/pivottable-pivotlayout.yaml

```diff
@@ -0,0 +1,322 @@
+order: 2
+id: excel-pivottable-pivotlayout
+name: PivotLayout
+description: Sets PivotTable layout settings through the PivotLayout.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); 
+        $("#change-layout").click(() => tryCatch(changeLayout)); 
+        $("#add-alt-text").click(() => tryCatch(addAltText)); 
+        $("#add-line-spacing").click(() => tryCatch(addLineSpacing)); 
+        $("#repeat-item-labels").click(() => tryCatch(repeatItemLabels)); 
+        $("#toggle-field-headers").click(() => tryCatch(toggleFieldHeaders)); 
+        $("#toggle-grand-totals").click(() => tryCatch(toggleGrandTotals)); 
+        $("#set-empty-cell-text").click(() => tryCatch(setEmptyCellText)); 
+        $("#toggle-fill-empty-cells").click(() => tryCatch(toggleFillEmptyCells)); 
+        $("#toggle-auto-format").click(() => tryCatch(toggleAutoFormat)); 
+        $("#toggle-preserve-formatting").click(() => tryCatch(togglePreserveFormatting));
+
+        async function changeLayout() {
+          await Excel.run(async (context) => {
+            // Change the PivotLayout.type to a new type.
+            const pivotTable = context.workbook.worksheets.getActiveWorksheet().pivotTables.getItem("Farm Sales");
+            pivotTable.layout.load("layoutType");
+            await context.sync();
+
+            // Cycle between the three layout types.
+            if (pivotTable.layout.layoutType === "Compact") {
+              pivotTable.layout.layoutType = "Outline";
+            } else if (pivotTable.layout.layoutType === "Outline") {
+              pivotTable.layout.layoutType = "Tabular";
+            } else {
+              pivotTable.layout.layoutType = "Compact";
+            }
+
+            await context.sync();
+            console.log("Pivot layout is now " + pivotTable.layout.layoutType);
+          });
+        }
+
+        async function addAltText() {
+          await Excel.run(async (context) => {
+            // Set the alt text for the PivotTable object.
```

Comment:
Should this comment say PivotLayout instead of PivotTable? We set the alt text for the entire PivotTable, but it's set with the PivotLayout class. Maybe something like "Use the PivotLayout class to set the alt text for the PivotTable."

Reply:
What's really happening is that the alt text is being set for the OArt on the worksheet. In this sense, the PivotTable is like a Chart or a Shape and I want to make the point that the alt text is associated with that floating shape.